projects/plugins/jetpack/_inc/lib/core-api/wpcom-endpoints/class-wpcom-rest-api-v2-endpoint-videopress.php

```diff
@@ -31,44 +31,52 @@ public function register_routes() {
 			$this->rest_base . '/meta',
 			array(
 				'args'                => array(
-					'id'             => array(
+					'id'              => array(
 						'description'       => __( 'The post id for the attachment.', 'jetpack' ),
 						'type'              => 'int',
 						'required'          => true,
 						'validate_callback' => function ( $param ) {
 							return is_numeric( $param );
 						},
 					),
-					'title'          => array(
+					'title'           => array(
 						'description'       => __( 'The title of the video.', 'jetpack' ),
 						'type'              => 'string',
 						'required'          => false,
 						'sanitize_callback' => 'sanitize_text_field',
 					),
-					'description'    => array(
+					'description'     => array(
 						'description'       => __( 'The description of the video.', 'jetpack' ),
 						'type'              => 'string',
 						'required'          => false,
 						'sanitize_callback' => 'sanitize_text_field',
 					),
-					'rating'         => array(
+					'rating'          => array(
 						'description'       => __( 'The video content rating. One of G, PG-13 or R-17', 'jetpack' ),
 						'type'              => 'string',
 						'required'          => false,
 						'sanitize_callback' => 'sanitize_text_field',
 					),
-					'display_embed'  => array(
+					'display_embed'   => array(
 						'description'       => __( 'Display the share menu in the player.', 'jetpack' ),
 						'type'              => 'boolean',
 						'required'          => false,
 						'sanitize_callback' => 'rest_sanitize_boolean',
 					),
-					'allow_download' => array(
+					'allow_download'  => array(
 						'description'       => __( 'Display download option and allow viewers to download this video', 'jetpack' ),
 						'type'              => 'boolean',
 						'required'          => false,
 						'sanitize_callback' => 'rest_sanitize_boolean',
 					),
+					'privacy_setting' => array(
+						'description'       => __( 'Requires the user to be a member of the site to view the video', 'jetpack' ),
```

Comment:
This might need some clarification

```suggestion
						'description'       => __( 'How to determine if the video should be public or private', 'jetpack' ),
```